### ISSUE TITLE
I can't see patients I just created

### ISSUE THREAD

**Issue by martinmodrak** [drwaseemsheikh/audiometry#7]
I created and saved two patients (both without using anything except the "Patient" tab). I am unable to find them in the database.

**Comment by drwaseemsheikh** [comment]
It doesn't make sense to save a patient record without any test conducted. Can you think of use case for that in a clinical setting? The software is behaving as expected.

**Comment by martinmodrak** [comment]
1. The issue persists even when I fill in some values in the "Pure Tone" tab. I am happy to send you some logs or anything else you might need to debug this, it is possible it is related to my computer setup (I am on Windows 10, English edition).
2. I believe there is a use case for documenting only symptoms and diagnosis ("Exam Information" box on the "Patient" tab) without conducting tests or for the case when the examination is for any reason interrupted before conducting a test, in which case I might not want to fill in the patient data again once the examination resumes.
3. If I fill in something, click "Save", the program tells me that "Patient record with account number 1 and date of exam 2/24/2020 12:30:11 PM written to the database." and then nothing is saved, it is definitely a bug. If for some reason the software cannot/should not save patient without any tests conducted, the message upon clicking "Save" should explicitly say something like "No tests conducted, patient data will NOT be saved".

**Comment by martinmodrak** [comment]
The issue persists after updating to latest version, even if I fill some values in the "Pure tone" tab.

**Comment by drwaseemsheikh** [comment]
I am unable to reproduce this bug. I will request Afron to ask another reviewer to test this issue and provide detailed steps of how the bug is introduced. Just saying that the issue persists which I can't replicate is not helpful. I am unable to reproduce this bug and when I save the patient data alone, I am able to save and restore it. So we are at an impasse and I would ask the editor to find a way out of this.

**Comment by martinmodrak** [comment]
I tried on a different PC (Windows 7, .NET Framework 4.7.02053) and I still can't save the patient. I followed exactly the steps you mentioned at https://github.com/openjournals/joss-reviews/issues/2016#issuecomment-633714396 . The confirmation dialog (patient saved) appears, but when I click "Open" the list of patients is empty. I would suggest you also try to install and run Audiometry on a few different PCs and see if any of them has the same issue. I reiterate that if you want any diagnostic info, I'll be happy to provide it.
